components/properties/Cargo.toml

```diff
@@ -35,9 +35,11 @@ all-features = true
 displaydoc = { version = "0.2.3", default-features = false }
 icu_codepointtrie = { version = "0.3.3", path = "../../utils/codepointtrie"}
 icu_provider = { version = "0.5", path = "../../provider/core", features = ["macros"] }
+icu_provider_fs = {version = "0.5.0", path = "../../provider/fs" }
 icu_uniset = { version = "0.4.1", path = "../../utils/uniset"}
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
 zerovec = { version = "0.6", path = "../../utils/zerovec", features = ["derive"] }
+unicode-bidi = { git = "https://github.com/servo/unicode-bidi", optional = true , deafault-features = false }
```

Comment:
i've made https://github.com/servo/unicode-bidi/pull/78 to get this published, since we have a working PR now and are close to landing it

Reply:
done